core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerImpl.java

```diff
@@ -80,6 +86,7 @@
    private TimeService timeService;
    private final List<CacheLoader> loaders = new ArrayList<>();
    private final List<CacheWriter> writers = new ArrayList<>();
+   private final List<CacheWriter> txWriters = new ArrayList<>();
```

Comment:
If the `writers` collection now only contains the non-tx writers, you should rename it to `nonTxWriters`.